.github/workflows/PR_reviewer.yml

```
@@ -11,81 +10,48 @@ on:
-        env:
-          GTP_TOKEN: ${{ secrets.CROSS_REPO_TOKEN }}
-        run: |
-          [ -f package.json ] && mv package.json package.json.xx
-          npm install openai@4.66.1 @octokit/rest @slack/web-api@7.5.0 slackify-markdown@4.4.0 @actions/github@6.0.0 @actions/core@1.10.1 node-fetch@cjs outdent@0.8.0
+      - name: Install Dependencies
+        working-directory: pr-reviewer/.github/scripts/pr-reviewer
+        run: npm install
 
       - name: Run PR Reviewer
-        if: steps.save_pr_description.outcome == 'failure' || steps.cache.outputs.cache-hit != 'true'
         uses: actions/github-script@v6
```

actionlint (1.7.4)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)